Todo-script.js

```diff
@@ -1,18 +1,60 @@
+const POPUP_ERROR = "<div class='error-popup'>";
+const POPOUP_TIME = 1500;
+const CL_ERROR = "Item not found";
+const CHILL = "You have nothing to do";
+let todoItems = [];
 $(document).ready(() => {
   const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
-  const POPOUP_TIME = 1500;
-  const POPUP_ERROR = "<div class='error-popup'>";
   const ERROR_MESSAGE = "You must enter something on To-do list";
+  const CLEAR_LIST_MESSAGE = "List is clear!";
+  const TODO_LIST = $(".todo-list");
+  const STORED_TODO_LIST = localStorage.getItem("todoList");
 
-  const todoList = $(".todo-list");
-  todoList.find("li").each((element) => $(element).append(SPAN_CLOSE));
+  if (!STORED_TODO_LIST) {
+    showError(CHILL);
+  } else {
+    todoItems = parseJson(STORED_TODO_LIST);
 
-  todoList.on("click", ".close", (event) => {
-    $(event.currentTarget).parent().toggle();
+    for (const item of todoItems) {
+      const LIST_ITEM = $("<li>").text(item.task).append(SPAN_CLOSE);
+      if (item.completed) {
+        LIST_ITEM.addClass("checked");
+      }
+      TODO_LIST.append(LIST_ITEM);
+    }
+  }
+
+  $(".todo-list")
+    .find("li")
+    .each(function () {
+      $(this).append(SPAN_CLOSE);
+    });
+
+  $(".todo-list").on("click", ".close", function (event) {
+    const PARENT_LI = $(event.currentTarget).closest("li");
+    const INDEX = PARENT_LI.index();
+
+    if (INDEX !== -1 < todoItems.length) {
+      todoItems.splice(INDEX, 1);
+      PARENT_LI.remove();
+
+      updateLocalStorage(todoItems);
+    } else {
+      showError(CL_ERROR);
+    }
   });
 
-  todoList.on("click", "li", (event) => {
-    $(event.currentTarget).toggleClass("checked");
+  $(".todo-list").on("click", "li", (event) => {
+    const CLICKED_ITEM = $(event.currentTarget);
```

Comment:
even though in theory the value in these variables doesnt change, but you dont have to name them with capital letter.  But it doesnt matter that much, just in the future, all the constants you define at the start of the file should be named with capital letters, but local variables inside the function shouldnt.